enigmatry-entry-blueprint-app/Enigmatry.Entry.Blueprint.App.csproj

```diff
@@ -1,55 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
-    <TypeScriptToolsVersion>4.4</TypeScriptToolsVersion>
-    <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
-    <TypeScriptCompileOnSaveEnabled>False</TypeScriptCompileOnSaveEnabled>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
     <AspNetCoreHostingModel>OutOfProcess</AspNetCoreHostingModel>
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Remove="dist\**" />
-    <Compile Remove="e2e\**" />
-    <Compile Remove="src\**" />
-    <Compile Remove="wwwroot\**" />
     <Content Remove="dist\**" />
+    <None Remove="dist\**" />
     <Content Remove="e2e\**" />
     <Content Remove="src\**" />
-    <Content Remove="wwwroot\**" />
-    <EmbeddedResource Remove="dist\**" />
-    <EmbeddedResource Remove="e2e\**" />
-    <EmbeddedResource Remove="src\**" />
-    <EmbeddedResource Remove="wwwroot\**" />
-    <None Remove="dist\**" />
-    <None Remove="e2e\**" />
     <None Remove="src\**" />
-    <None Remove="wwwroot\**" />
-    <None Remove=".browserslistrc" />
+    <Content Remove="wwwroot\**" />
+    <None Remove=".eslintignore" />
     <None Remove=".gitignore" />
-    <None Remove=".npmrc" />
-    <None Remove="angular_material.txt" />
     <None Remove="karma.conf.js" />
-    <None Remove="tsconfig.json" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <TypeScriptCompile Remove="*" />
-    <TypeScriptCompile Remove="e2e\**" />
-    <TypeScriptCompile Remove="node_modules\**" />
-    <TypeScriptCompile Remove="src\**" />
```

Comment:
Did you check if this causes any issue with R# or VS. We do not want these files to be scaned (unless we want to start developing Angular through VS or Rider)